R/sim_linelist.R

```diff
@@ -56,9 +56,16 @@
 #' confirmed case and `NA` otherwise for each case in the line list.
 #' Default is `TRUE`. Ct refers to the Cycle threshold from a Real-time
 #' PCR or quantitative PCR (qPCR).
-#' @param min_outbreak_size A single `numeric` defining the minimum chain size
-#' for the simulated outbreak. Default is `10`. This can be increased when the
-#' function should simulate a larger outbreak.
+#' @param outbreak_size A `numeric` vector of length 2 defining the minimum and
+#' the maximum number of infected individuals for the simulated outbreak.
+#' Default is `c(10, 1e5)`, so the minimum outbreak size is 10 infected
+#' individuals, and the maximum outbreak size is 100,000 infected individuals.
```

Comment:
Maybe making this smaller by default would be helpful to reduce the time that it takes to simulate linelists and make it less likely to get stuck for 10+ mins like it was happening to me yesterday? I don't think many people would want to simulate 100K cases anyway, maybe 10K maximum by default?

Reply:
I agree that most users will not need to simulate >100,000 cases unless they are wanting emulate an epidemic like Ebola 2014-16 or a pandemic like Covid-19. I've lower the maximum to 10,000 in d49c506d9bcb44929252114f3f3c2878e3ae3ba4.